pkg/virt-launcher/virtwrap/network/podinterface.go

```diff
@@ -1184,9 +1187,9 @@ func (b *MasqueradeBindMechanism) getGatewayByProtocol(proto iptables.Protocol)
 
 func (b *MasqueradeBindMechanism) getDhcpConfigIpByProtocol(proto iptables.Protocol) string {
```

Comment:
I know you didn't add it, but now you remove the dhcpConfig from here this method name is even more confusing.
I suggest `geVmIfaceIpByProtocol`.